src/sb-util.ts

```diff
@@ -162,10 +168,37 @@ export class BlockCollection implements Queryable {
 		storage.set(this, blocks);
 	}
 
-	query() {
-		throw new Error('BlockCollection queryable not implemented yet!')
+	first() {
```

Comment:
What about?

```js
first() {
    return first(this[Symbol.iterator]());
}
```

Reply:
Covers returning a Block or null since first does that.

Reply:
Could make an iterable version of `first` or make it support both, and be able to `first(this)`.